dozer-cli/src/pipeline/dummy_sink.rs

```diff
@@ -51,7 +55,28 @@ impl Sink for DummySink {
         Ok(())
     }
 
-    fn on_source_snapshotting_done(&mut self, _connection_name: String) -> Result<(), BoxedError> {
+    fn on_source_snapshotting_started(
+        &mut self,
+        connection_name: String,
+    ) -> Result<(), BoxedError> {
+        self.snapshotting_started_instant
+            .insert(connection_name, Instant::now());
+        Ok(())
+    }
+
+    fn on_source_snapshotting_done(&mut self, connection_name: String) -> Result<(), BoxedError> {
+        if let Some(started_instant) = self.snapshotting_started_instant.remove(&connection_name) {
+            println!(
```

Comment:
Should `println` be replaced with one of the log macros?

Reply:
Sure